### PR TITLE
[IMP] account: missing translations

### PR DESCRIPTION
**Pull request by h4818**
Numerous terms were missing from the translation template file.

### CODE REVIEW

File: addons/account/i18n/account.pot [hunk 1]
```diff
@@ -13523,10 +13565,12 @@ msgid "The company this distribution line belongs to."
 msgstr ""
 
 #. module: account
+#. odoo-python
 #: code:addons/account/models/partner.py:0
 #, python-format
 msgid ""
-"The country of the foreign VAT number could not be detected. Please assign a country to the fiscal position or set a country group"
```
Comment (ticodoo) [inline addons/account/i18n/account.pot:13529]: looks like this was added to the wrong pot file somehow :facepalm: The string is in `base_vat`, but isn't in its pot file, https://github.com/odoo/odoo/pull/173976

Not sure if you want to re-export that one too

File: addons/account/i18n/account.pot [hunk 2]
```diff
@@ -492,11 +491,6 @@ msgid ""
 "#704A66;'>%(date)s</b>"
 msgstr ""
 
-#. module: account
-#: model_terms:ir.ui.view,arch_db:account.view_account_payment_register_form
-msgid "<b>Early Payment Discount of %(payment_reference) has been applied.</b>"
```
Comment (ticodoo) [inline addons/account/i18n/account.pot:497]: This is weird. Double checking if this string should be correctly exported somehow

File: addons/account/i18n/account.pot [hunk 3]
```diff
@@ -6,8 +6,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Odoo Server 17.0+e\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2024-07-24 12:25+0000\n"
-"PO-Revision-Date: 2024-07-24 12:25+0000\n"
```
Comment (ticodoo) [inline addons/account/i18n/account.pot:10]: no need to include the date changes since it will cause merge conflicts

**Review by ticodoo — COMMENTED** [review]
I will get back to you about the weird not-exported term, but once the date change is reverted I can r+

File: addons/account/i18n/account.pot (hunk 2 above)
Comment (ticodoo) [reply addons/account/i18n/account.pot:497]: Ah false alarm, it did export it as 2 separate terms as expected. Sorry I am blind :see_no_evil:

File: addons/account/i18n/account.pot (hunk 1 above)
Comment (h4818) [reply addons/account/i18n/account.pot:13529]: It's there https://github.com/odoo/odoo/blob/17.0/addons/base_vat/i18n/base_vat.pot#L207

File: addons/account/i18n/account.pot (hunk 2 above)
Comment (h4818) [reply addons/account/i18n/account.pot:497]: I think it is exported properly as 2 strings:
`Early Payment Discount of`
`has been applied.`

File: addons/account/i18n/account.pot (hunk 1 above)
Comment (ticodoo) [reply addons/account/i18n/account.pot:13529]: Ah oops, sorry I missed that the strings are slightly different between the 2 modules

**Review by ticodoo — COMMENTED** [review]
Thanks!

@robodoo r+

**Comment by ticodoo** [comment]
@robodoo up to saas-17.4

**Pull request closed without merging**